docs/differences-from-v1.md

```diff
@@ -4,41 +4,80 @@ Upgrading from FOSSA CLI 1.x to 2.x is a major breaking change. For most users,
+    - [Remove calls to `fossa init`](#remove-calls-to-fossa-init)
+    - [Migrate your .fossa.yml file](#migrate-your-fossayml-file)
+    - [Migrate "archive upload" targets](#migrate-archive-upload-targets)
+    - [Getting help with your migration](#getting-help-with-your-migration)
+    - [Migrate "fossa report" commands](#migrate-fossa-report-commands)
+  - [Frequently Asked Questions](#frequently-asked-questions)
+      - [Until when CLI v1 will be supported by FOSSA?](#until-when-cli-v1-will-be-supported-by-fossa)
+      - [I'm getting a poor result with CLI v2 compared to CLI v1.](#im-getting-a-poor-result-with-cli-v2-compared-to-cli-v1)
+      - [How do I run only a specific analyzer for my project?](#how-do-i-run-only-a-specific-analyzer-for-my-project)
+    - [How do I identify which version of CLI I have installed?](#how-do-i-identify-which-version-of-cli-i-have-installed)
+      - [I'm running into an error - how do I debug?](#im-running-into-an-error---how-do-i-debug)
 
 ## What's new in FOSSA 2.x?
 
 Our focus in CLI 2.x is making FOSSA integrations easier to deploy by reducing the amount of configuration needed by users. Our goal is to get as close to turnkey deployment on as many build systems and codebases as possible.
 
-### "Modules" are now "analysis targets"
-
-In 1.x, the CLI was configured by "modules", which were entrypoints to user programs that the CLI should analyze.
+### New Build Manager Supports
 
-In 2.x, we've renamed "modules" to "analysis targets", and simplified how analysis targets are defined.
+FOSSA 2.x supports following new build managers and languages:
 
-We renamed analysis targets to avoid confusing naming conflicts with existing language-specific "modules". Analysis targets are no longer defined by multiple, confusingly similar fields (e.g. "Dir" vs. "Path"), and are now simple `(type, target)` tuples.
+- [Swift Package Manager (for swift)](strategies/ios/swift.md)
+- [Pub (for flutter and dart)](strategies/dart.md)
+- [Poetry (for Python)](strategies/python.md)
+- [Mix (for Elixir)](strategies/elixir.md)
+- [Fortran Package Manager (for Fortran)](strategies/fortran.md)
 
 ### Automatic analysis target discovery
 
-FOSSA CLI 2.x now does automatic analysis target discovery when you run `fossa analyze`.
+FOSSA CLI 2.x now does automatic analysis target discovery when you run `fossa analyze`. This means that `fossa init` is no longer required. `fossa init` is now a no-op that emits a warning, and will be removed in a future release.
 
-This means that `fossa init` is no longer required. `fossa init` is now a no-op that emits a warning, and will be removed in a future release.
+### New fossa-deps configuration support
 
-### Automatic analysis target strategy selection
+With .fossa-deps.yml file, 2.x supports:
+
+- License scanning vendor dependencies 
+- Manually specifying dependency for analysis by its source code archive web address (e.g. https://my-deps-source/v1.zip)
```

Comment:
> its source code archive web address
This sounds really weird to me. Something like the following might be better:
"Analyzing archives that are located at a specific web address"